OXPlugin.cs

```diff
@@ -387,10 +387,19 @@ public HTTPResponse GetResponse(HTTPRequest request)
                 {
                     if (parameters.ContainsKey("username") && parameters["username"] != "" && parameters.ContainsKey("password") && parameters["password"] != "")
                     {
-                        AddPlayer(parameters["username"].Trim(), parameters["password"].Trim());
-                        sb.Append("<h2>Player added successfully</h2>");
-                        sb.Append(String.Format("Please note your login is <b>{0}</b> and password is <b>{1}</b>. <br />", parameters["username"], parameters["password"]));
-                        sb.Append(String.Format("<a href=\"/ox?username={0}\">Click here to go back to home page</a>", parameters["username"]));
+                        Player player = GetPlayerByUserName(parameters["username"]);
```

Comment:
player name is case-sensitive or case-insensitive?

Reply:
Now, it's case-insensitive but I have case-sensitive checking in new commit. How I add it to merge ?